lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -40,22 +86,38 @@ class WP_Font_Collection {
 	 */
 	public function __construct( $config ) {
 		if ( empty( $config ) || ! is_array( $config ) ) {
-			throw new Exception( 'Font Collection config options is required as a non-empty array.' );
+			throw new Exception( 'Font Collection config options are required as a non-empty array.' );
```

Comment:
I think using `_doing_it_wrong` is preferred here (and below), as the standard way to indicate to developers  something is being called wrong or used incorrectly.

Reply:
:+1: changed in this commit.